### PR TITLE
fix - change password validators moved to wrapper, because of redux-form limitation

### PR DESCRIPTION
**Pull request by tony-blockchain**
## Description
Redux form bug does not allow to create scopes upon rendering https://github.com/erikras/redux-form/issues/3405
Did not investigate why, but perhaps they just compare prop equality
For a fix I've moved validators from template to the form wrapping component.

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
1. Enter incorrect password in the "current password" field. **After the first blur** it should have red border when wrong password is entered.
2. Enter mismatching password to the password confirmation. It should turn red after the first blur as well.

## Code Checklist
- [+] Code compiles successfully (verified via `yarn start`)
- [+] No lint issues exist (verified via `yarn lint`)
- [+] New and existing unit tests pass (verified via `yarn test`)
- [+] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js [hunk 1]
```diff
@@ -25,11 +25,19 @@ class SettingsContainer extends React.PureComponent {
...
 
+  validateCurrentPassword = (value) =>
+    value === this.props.currentWalletPassword ? undefined : 'Incorrect password'
+
+  validatePasswordConfirmation = (value, allValues) =>
+    (value === allValues.newPassword) ? undefined : 'Passwords do not match'
+
   render () {
-    const { ui, ...rest } = this.props
+    const { ui, currentWalletPassword, ...rest } = this.props
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js]: I don't think this is used?

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js]: It's to remove it from being passed onwards

File: packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js [hunk 2]
```diff
@@ -25,11 +25,19 @@ class SettingsContainer extends React.PureComponent {
     this.props.updateUI({ updateToggled: !this.props.ui.updateToggled })
   }
 
+  validateCurrentPassword = (value) =>
+    value === this.props.currentWalletPassword ? undefined : 'Incorrect password'
+
+  validatePasswordConfirmation = (value, allValues) =>
+    (value === allValues.newPassword) ? undefined : 'Passwords do not match'
+
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js]: It seems we're keeping many of our validators in `services/FormHelper`, could you move those there for consistency and translations?

Comment (tony-blockchain) [reply packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js]: Sure

**Comment by plondon** [comment]
Closing in favor of https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/652

**Pull request closed without merging**